pkg/types/ccipocr3/plugin_commit_types.go

```diff
@@ -66,12 +66,22 @@ type PriceUpdates struct {
 	GasPriceUpdates   []GasPriceChain `json:"gasPriceUpdates"`
 }
 
+type CommitReportArgs struct {
+	ReportContext [2][32]byte
+	Report        []byte
+	Rs            [][32]byte
+	Ss            [][32]byte
+	RawVs         [32]byte
+	Info          CommitReportInfo
+}
```

Comment:
Please remove this. It's OCR/CW code for the transmitter.

Reply:
Is there some other common place we can add this outside of chainlink core? I want to be able to import this into `chainlink-solana` to run some tests

Reply:
What kind of tests? All integration tests are currently in chainlink. If this is transmitter related, it should be in chainlink and the test should be there as well

Reply:
I wrote some tests specifically to test the Solana ChainWriter's ability to receive the contract transmitter args and encode / transform them for execution on chain https://github.com/smartcontractkit/chainlink-solana/blob/da75f6d7734685f31e757b191725a77cf0094120/pkg/solana/chainwriter/chain_writer_test.go#L697-L752

Reply:
I'd like to see a generic type as part of the CW that we could use:
```
type ChainWriterOCRArgs struct {
	ReportContext [2][32]byte
	Report        []byte
	Rs            [][32]byte // omit empty?
	Ss            [][32]byte // omit empty?
	RawVs         [32]byte // omit empty?
	// Info is a special field used by CW to dynamically accept chain specific data.
	Info          any
}
```

Reply:
Ok, removed this from this PR